### DISCUSSION TITLE
How to refresh an entry in the cache?

### DISCUSSION THREAD

**Discussion by nggonzalez** [bvaughn/suspense#40] [Q&A] [answered]
Hi, what is the best way to refresh an entry in the cache? 

My specific use case is that I want to refetch the data when the user navigates back to a particular screen.

So far I've thought of the following approaches, but could use some guidance:
- `useCacheMutation` and follow the steps here https://github.com/bvaughn/suspense/discussions/15
- Evict the entry

Thanks for the help!

**Comment by bvaughn** [comment, marked as answer]
`useCacheMutation` is the only recommended way of mutating a cache.

> Evict the entry

This will likely not do anything in terms of scheduling updates or letting React know that the previously cached value is now stale.

**Reply by nggonzalez** [reply]
Thanks for the quick answers, Brian! I appreciate the help :)
